content/blog/2018-08-28-birds-ocr.md

```diff
@@ -0,0 +1,411 @@
+---
+title: "What's this bird? Classify old natural history drawings with R"
+slug: birds-ocr
+authors:
+  - name: Maëlle Salmon
+    url: https://masalmon.eu/
+date: 2018-08-28
+preface: The blog post series corresponds to the material for a talk Maëlle will give at the [Animal Movement Analysis summer school in Radolfzell, Germany on September the 12th](http://animove.org/animove-2019-evening-keynotes/), in a Max Plant Institute of Ornithology.
+tags:
+- magick
+- tesseract
+- cld2
+- cld3
+- taxize
+- birder
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+In this new post, we’re taking a break from modern birding data in [our
+birder’s series](https://ropensci.org/tags/birder)… let’s explore
+gorgeous drawings from a natural history collection! Armed with
+rOpenSci’s packages binding powerful C++ libraries and open taxonomy
+data, how much information can we automatically extract from images?
+Maybe not much, but we’ll at least have explored image manipulation,
+optical character recognition, language detection, taxonomic name
```

Comment:
add (OCR) since abbrev is used in heading below without definition
search doc and use OCR where appropriate

Reply:
good point